package/qt_widgets/example.py

```diff
@@ -0,0 +1,115 @@
+import sys, numpy as np
+sys.path.append("C:\\surface-view\\package")
```

Comment:
delete this line

Reply:
I agree

Reply:
I'm not removing it because it's a test use case, not a prototype or anything